src/strings/PowerFxResources.en-US.resx

```diff
@@ -4846,4 +4846,11 @@
     <value>The 'User-defined types' experimental feature is not enabled.</value>
     <comment>Error message returned when user uses User-defined types with the User-defined types feature flag turned off.</comment>
   </data>
+  <data name="ErrUserDefinedTypeIncorrectSyntax" xml:space="preserve">
+    <value>Incorrect syntax `=` used to define User-defined type.</value>
```

Comment:
>`=` [](http://example.com/codeflow?start=28&length=3)

Suggestion: use normal single quotes, not back ticks (consistent with other places in this file)